backend/src/services/identity-access-token/identity-access-token-service.ts

```diff
@@ -107,15 +109,15 @@ export const identityAccessTokenServiceFactory = ({
 
       if (currentDate > expirationDate) {
         await identityAccessTokenDAL.deleteById(identityAccessToken.id);
-        throw new UnauthorizedError({
+        throw new ForbiddenRequestError({
```

Comment:
401 Unauthorized: when the request lacks valid authentication credentials or the provided credentials have become invalid (expired, etc)

403 Forbidden: when the client is authenticated but lacks the necessary permissions for the requested action

This should also be UnauthorizedError. If you can double check this file for this, that would be good